src/libServer/WebsocketServer.cpp

```diff
@@ -63,6 +63,7 @@ bool WebsocketServer::start() {
   clean();
   // Initialising websocketserver
   m_server.init_asio();
+  m_server.set_reuse_addr(true);
```

Comment:
@kaikawaliu can u check this if this makes any difference to the core protocol. In my opinion, it does not.

Reply:
Yup I think it's good to have